src/hal_simplicity.rs

```diff
@@ -93,11 +93,11 @@ impl<J: Jet> Program<J> {
 }
 
 // Stolen from simplicity-webide
```

Comment:
Probably a good idea to change this comment and explain where the key came from. eg. include [this link](https://github.com/BlockstreamResearch/secp256k1-zkp/blob/11af7015de624b010424273be3d91f117f172c82/src/modules/rangeproof/main_impl.h#L16)